scripts/west_commands/tests/test_bossac.py

```diff
@@ -16,27 +16,39 @@
     pytest.skip("skipping Linux-only bossac tests", allow_module_level=True)
 
 TEST_BOSSAC_PORT = 'test-bossac-serial'
+TEST_BOSSAC_SPEED = '115200'
```

Comment:
This should differ from the default value; the point of this group of constants is to make sure the command line options are respected.

Reply:
fixed!